proof/invariant-abstract/SchedContextInv_AI.thy

```diff
@@ -1566,10 +1566,12 @@ lemma handle_timeout_Timeout_invs:
    apply (clarsimp simp: pred_tcb_at_def obj_at_def dest!: get_tcb_SomeD)
    apply (drule invs_iflive)
    apply (drule (1) if_live_then_nonz_capD2)
-    apply (fastforce simp: live_def split: )
+    apply (fastforce simp: live_def)
    apply clarsimp
-  apply (rule_tac x=4 in exI)
-  apply (auto simp: tcb_cnode_map_def caps_of_state_tcb_index_trans)
+  apply (rule_tac P="\<exists>n. P n" and P'="P 4" and Q=Q and Q'=Q for P Q in conj_forward)
```

Comment:
Not a blocking issue at all, but out of curiosity do you remember where the `4` comes from in this proof?

Reply:
Refers to the timeout handler, which is in the 4th slot